controller.xql

```diff
@@ -22,22 +22,21 @@ if ($exist:path eq '') then
     <dispatch xmlns="http://exist.sourceforge.net/NS/exist">
         <redirect url="{concat(request:get-uri(), '/')}"/>
     </dispatch>
-    
+
 else if ($exist:path = "/") then
     (: forward root path to index.xql :)
     <dispatch xmlns="http://exist.sourceforge.net/NS/exist">
         if(request:get-uri() eq "/exist/apps/dashboard/" and request:get-header("X-Forwarded-URI") eq "/apps/dashboard/") then
             <redirect url="/apps/dashboard/index.html"/>
         else
             <redirect url="index.html"/>
-    }
```

Comment:
Looks like it was introduced in https://github.com/eXist-db/dashboard/commit/af4cc34b8622a75b5c35289f32e4fab62ccb0207 - but luckily this didn't make it into a release. Perhaps we should follow @JonathanRowell's lead and begin to institute some basic linting of our XQuery files, ideally automated via CI?